package.json

```diff
@@ -4,13 +4,26 @@
   "license": "MIT",
   "type": "module",
   "author": "Thinkmill Labs Pty Ltd",
-  "repository": "https://github.com/thinkmill/manylicenses",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thinkmill/manylicenses.git"
+  },
+  "files": [
+    "lib/*"
+  ],
```

Comment:
Only include `lib` in the package, plus any files npm includes by default like package.json, LICENSE etc.